Userland/Libraries/LibJS/MarkupGenerator.cpp

```diff
@@ -145,11 +145,14 @@ void MarkupGenerator::date_to_html(Object const& date, StringBuilder& html_outpu
 
 void MarkupGenerator::trace_to_html(TracebackFrame const& traceback_frame, StringBuilder& html_output, HashTable<Object*>&)
 {
-    auto const& function_name = escape_html_entities(traceback_frame.function_name.is_empty() ? "<anonymous>" : traceback_frame.function_name);
-    auto const& source_range = traceback_frame.source_range;
-    auto const& last_slash_index = source_range.filename.find_last('/');
-    auto const& filename = escape_html_entities(last_slash_index.has_value() ? source_range.filename.substring_view(*last_slash_index + 1) : source_range.filename);
-    auto const& trace = String::formatted("at {} ({}:{}:{})", function_name, filename, source_range.start.line, source_range.start.column);
+    auto function_name = escape_html_entities(traceback_frame.function_name);
```

Comment:
Are functions guaranteed to have a non-empty function? The previous iteration of this code has `traceback_frame.function_name.is_empty() ? "<anonymous>" : traceback_frame.function_name`.

Is this no longer needed?

Reply:
I wasn't able to get this this "<anonymous>" string while testing. LibJS replaces empty function names with the "<unknown>" string:
https://github.com/SerenityOS/serenity/blob/dc71e1e2640b734f8bd0cdbb9a10168ca3ba7d53/Userland/Libraries/LibJS/Runtime/Error.cpp#L63

Reply:
Ah I didn't know that, seems fine then.